py/obj.h

```diff
@@ -465,8 +465,12 @@ struct _mp_obj_type_t {
     // A type is an object so must start with this entry, which points to mp_type_type.
     mp_obj_base_t base;
 
-    // The name of this type.
-    qstr name;
+    // Flags associated with this type.
+    #define MP_OBJ_TYPE_FLAG_xyz (0x0001)
+    uint16_t flags;
+
+    // The name of this type, a qstr.
+    uint16_t name;
```

Comment:
Just to doublecheck: a qstr is a size_t but it's ok to cast it to uint16_t right?